apps/financial-aid/web-veita/src/components/ModalTypes/AcceptModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+
+import {
+  InputModal,
+  NumberInput,
+} from '@island.is/financial-aid-web/veita/src/components'
+import { Input, Box } from '@island.is/island-ui/core'
+
+interface Props {
+  onCancel: (event: React.MouseEvent<HTMLButtonElement>) => void
+  onSaveApplication: (amount: number) => void
+  isModalVisable: boolean
+}
+
+const AcceptModal = ({
+  onCancel,
+  onSaveApplication,
+  isModalVisable,
+}: Props) => {
+  const maximumInputLength = 6
+  const [amount, setAmount] = useState<number>(0)
+
+  return (
+    <InputModal
+      headline="Umsóknin þín er samþykkt og áætlun er tilbúin"
+      onCancel={onCancel}
+      onSubmit={() => onSaveApplication(amount)}
```

Comment:
this should be:
`onSubmit={(amount) => onSaveApplication(amount)}`

Since amount is in state.